### DISCUSSION TITLE
Migrate the official DIscord server to Revolt

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [godotengine/godot-proposals#7051] [General Discussions]
Why? To put it short, Revolt is open source, Discord isn't.

**Comment by AThousandShips** [comment]
How does it compare feature wise? How easy is migrating?

I'm all for open source, and I use it whenever I can, but it doesn't always justify moving to a less competent platform

**Reply by RustoMCSpit** [reply]
> How does it compare feature wise? How easy is migrating?
> 
> I'm all for open source, and I use it whenever I can, but it doesn't always justify moving to a less competent platform

It literally is just DIscord but with less bots available as of moment, try it out. It isn't lacking in features or stability

**Reply by Zireael07** [reply]
https://itsfoss.com/revolt/ seems to indicate no MacOS support yet? So it's not "just Discord"... (Personally I'm on Linux but I know many Godot users are on Macs)

**Reply by RustoMCSpit** [reply]
> https://itsfoss.com/revolt/ seems to indicate no MacOS support yet? So it's not "just Discord"... (Personally I'm on Linux but I know many Godot users are on Macs)

Using LInux too, didn't realise there wasn't Mac yet.

**Reply by AThousandShips** [reply]
Also it doesn't look like there's any phone/tablet version of this either, which is pretty important for a lot of users, who might not even have a computer and still want to engage with the community

**Reply by Zireael07** [reply]
@AThousandShips Seconding, I have taken to looking at Discord from phone/tablet when on the go...

**Reply by sairam4123** [reply]
Revolt is available on android it seems "Revolt Chat"

**Reply by Deozaan** [reply]
I've never heard of Revolt until now but that article appears to be out of date. It's available on Android, iOS, Windows, Linux, MacOS, and web. https://revolt.chat/download

**Comment by RustoMCSpit** [comment]
https://rvlt.gg/qNcJpatE

**Comment by AThousandShips** [comment]
CC @coppolaemilio

**Reply by coppolaemilio** [reply]
People should create a Revolt server if they want to :) 
If they follow the CoC, it is active, and it is properly moderated, we should be able to include a link on our godotengine.org/community/ page. That's as official as anything can get really.
